api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -80,23 +82,24 @@ message ExtAuthz {
   // Clears the route cache in order to allow the external authorization service to correctly affect
   // routing decisions. The filter clears all cached routes when:
```

Comment:
Converting this to bullet points loses the implied "and" between the criteria (I assume this was the original intent, but fell free to validate this).
```suggestion
  // routing decisions. The filter clears all cached routes when all of the following holds:
```